### ISSUE TITLE
Bock Paper - Number of Neurons

### ISSUE THREAD

**Issue by adjordan** [Statistical-Connectomics-Sp15/intro#111]
If they only had 14 neurons, can this be considered statistically significant? I may have missed this in class, I went to the bathroom.

**Comment by itsthefedora** [comment]
I think they did an analysis of the statistical power of their methodology in the paper, that showed the results were "significant" under their assumptions.

... However, those assumptions included that samples of the differences between neurons' tuning are iid, which is pretty dumb, since if you have three tuned neurons going to one interneuron, the three sampled differences from that setup are *clearly* not independent. So ... basically, no one knows.

**Comment by whock** [comment]
Yeah claiming significance is tough with such a small n, but I thought it was equally about a proof-of-concept as it was about larger biological inferences. Like Joshua said in lecture, hadn't this not been done in a really long time and even then it was a lot more tedious before? Maybe the authors are hoping more groups will use their technique and get results that are more convincing. Also the graph they observed (provided their methods were sound, actually worked, etc), is 'real' in the sense that it's an example of actual orientation-selective neurons converging onto an inhibitory neuron. So maybe they'd argue it's not about statistical significance as much as it's about a case study of what neurons can (at least sometimes) do.

**Comment by willgray** [comment]
I think the enduring impact of this paper is the method - this is in many ways the first "real" EM connectomics paper, and as of May 2015, still the largest publicly available dataset.  In particular, integrating functional and anatomical measures successfully on the same sample was pretty amazing.  IMHO.
